visualization.shadertoy/resources/presets.json.md

```diff
@@ -0,0 +1,39 @@
+# User definable shader list within JSON file
```

Comment:
This whole section has been reviewed by @howie-f , so refer to this commit for changes:

https://github.com/howie-f/visualization.shadertoy/commit/dfc7c92b1d8eb784cee00837e000db985797adf2